lib/console_web/views/gateway_view.ex

```diff
@@ -23,6 +23,13 @@ defmodule ConsoleWeb.GatewayView do
     |> append_events(gateway.events)
   end
 
+  def render("gateway_register.json", %{tx: tx, signature: signature}) do
```

Comment:
this should probably be within the `views/router/` scope in order to keep this separate from any user-facing API